src/org/omegat/Bundle.properties

```diff
@@ -395,6 +395,12 @@ TF_MENU_GOTO_SELECTED_MATCH_SOURCE=Source of Selected &Match
 TF_MENU_GOTO_FORWARD_IN_HISTORY=&Forward in History
 TF_MENU_GOTO_BACK_IN_HISTORY=&Back in History
 
+TF_MENU_GOTO_X_SUBMENU=Auto-Populated Segments
+TF_MENU_GOTO_NEXT_XAUTO=Next Segment with xAUTO
+TF_MENU_GOTO_PREV_XAUTO=Previous Segment with xAUTO
+TF_MENU_GOTO_NEXT_XENFORCED=Next Segment with xENFORCED
+TF_MENU_GOTO_PREV_XENFORCED=Previous Segment with xENFORCED
```

Comment:
Are these have mnemonics to access from keyboard?
I have a question because these don't have shortcut key.

Reply:
Good point, I'll add mnemonics for sure. Thank you, Hiroshi!

Reply:
Done!

Reply:
Mnemonics added: [f8c4577](https://github.com/omegat-org/omegat/pull/370/commits/f8c457732e09c4aeb680be6a7077ea35a927ebdd)

Reply:
As to providing shortcuts, I'm not sure. We do have some menu items without shortcuts. Users can add them if they want to.

Reply:
I think it is better to have them. Adding shortcuts is not trivial and the functions are useful.

The functions that do not have shortcuts are functions that are generally used once in a while. In the Go to menu, we can filter on segments having notes so having a shortcut for "next note/previous note" is not really necessary.

(As far as I am concerned, the "search in dictionary" should have one too, for ex.)

Reply:
It creates a UX hierarchy between "auto" only 2 keys (C+A) and "enforce" with 3 keys (C+A+S).

Sometimes, the Shift is used to create the "reverse" action. So that would be Forward=C+A / Backward = C+A+S.

What would you think of that ?

Reply:
Committed the shortcuts as suggested (with Shift to revert direction).
If everyone is happy, maybe this one could be merged?